enterprise/models.py

```diff
@@ -2508,3 +2508,44 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class EnterpriseAnalyticsUser(TimeStampedModel):
+    """
+    Model for analytics users.
+
+     .. no_pii:
+    """
+    enterprise_customer_user = models.ForeignKey(
+        EnterpriseCustomerUser,
+        blank=False,
```

Comment:
I think by default blank and null is set to False, so no need to explicitly set it

Reply:
no harm in being explicit!